boards/posix/native_posix/doc/board.rst

```diff
@@ -159,7 +159,16 @@ Run the zephyr.exe executable as you would any other Linux console application.
 This executable accepts several command line options depending on the
 compilation configuration.
 You can run it with the ``--help`` command line switch to get a list of
-available options.
+available options::
+
+   $ zephyr/zephyr.exe --help
```

Comment:
Would it make sense to add a new  "command line options" subsection after "Running". List the options them there, noting that seed is optional. And replace the description of the `--seed` option from the "Peripherals" section with a reference to this new section?